cmd/orphan-finder/main.go

```diff
@@ -112,7 +112,7 @@ func parseLogLine(sa certificateStorage, logger blog.Logger, line string) (found
 		logger.AuditErr(fmt.Sprintf("Couldn't parse regID: %s, [%s]", err, line))
 		return true, false
 	}
-	_, err = sa.AddCertificate(ctx, der, int64(regID))
+	_, err = sa.AddCertificate(ctx, der, int64(regID), nil) // XXX: probably should do OCSP here too...?
```

Comment:
This is another case that makes it important to maintain the "new certificates" tick in ocsp-updater. Could you update the comment there to indicate that?